docs/installation.md

```diff
@@ -1,12 +1,14 @@
 # Installation
 
-Replace `{TRIVY_VERSION}` with the latest released version of Trivy. You can find the latest releases on this page: https://github.com/aquasecurity/trivy/releases
+!!! note
+    Replace `{TRIVY_VERSION}` with the latest released version of Trivy.
```

Comment:
If you publish docs for each release of Trivy, you could use macros plugin and define the tag and use it in installation scripts ``{{ var.trivy_version }}``

Reply:
I introduced `{{ git.tag }}`. I know it doesn't work for the `dev` release since it doesn't have git tags, but it is a document for developers. It is enough to support versioned documents at the moment.